__tests__/components/skeletons/Dashboard.test.js

```diff
@@ -10,7 +10,6 @@ describe('Dashboard', () => {
     render(<Dashboard sections={1} />)
 
     const box = screen.getByTestId('dashboard-skeleton')
-    console.log(box)
```

Comment:
remove log

Reply:
Yes that's what I did.

Reply:
I am embarrassed. I will learn to read :)